plugin/storage/integration/domain_trace_compare_test.go

```diff
@@ -50,13 +50,14 @@ func CompareTraces(t *testing.T, expected *model.Trace, actual *model.Trace) {
 	model.SortTrace(expected)
 	model.SortTrace(actual)
 	checkSize(t, expected, actual)
-	if !assert.EqualValues(t, expected, actual) {
-		for _, err := range pretty.Diff(expected, actual) {
-			t.Log(err)
-		}
+
+	diff := pretty.Diff(expected, actual)
+	if len(diff) > 0 {
+		t.Logf("Expected and actual differ: %v\n", diff)
```

Comment:
when there are many differences, this doesn't print them in easily readable way. I prefer the loop as before.